docs/graphql.md

```diff
@@ -0,0 +1,68 @@
+# Using GraphQL
+
+\_Note: If testing this locally, you must have telescope running locally as well. Please refer to the documentation [here](./environment-setup.md)
+
+# Adding/Modifying Resolvers for GraphQL
+
+1. Our resolvers can be found in [here](../src/backend/web/graphql/index.js) under module.exports.resolvers.
+
+## Step-by-step Guide
+
+1. To access GraphQL, append /grpahql to the end of your local app URI. For example if your Telescope app is running at http://localhost:3000, you can access GraphQL for telescope at http://localhost:3000/graphql
+
+2. A list of available resolvers for Telescope can be found in src/backend/web/graphql/index.js under module.exports.resolvers. The list of available resolvers may change over time.
+
+3. We will use the getPosts query, this query has two parameters page and perPage. For example, if a user wanted to see the results of page 10 and have 30 results per page, he/she would type the following:
+
+```
+{
+  getPosts(page: 10, perPage: 30) {
+
+  }
+}
+```
+
+This query won't run, because we also need to specify query fields. To see some available query fields type q within the getPosts() block:
+
+```
+{
+  getPosts(page: 10, perPage: 30) {
+    q
```

Comment:
I don't think this is helpful, since `q` isn't a valid field name.  Should we use `author` or the like?